README.md

```diff
@@ -26,7 +26,7 @@ see the [release notes](./documentation/release_notes.markdown).
 
 All versions of Puppet Server depend on at least Puppet 3.7.3, and since
 version 2.3 it depends on [Puppet Agent
-1.4.0](https://docs.puppet.com/puppet/4.4/reference/about_agent.html) or newer,
+1.4.0](https://docs.puppet.com/puppet/4.4/about_agent.html) or newer,
```

Comment:
This feels like a weird history lesson to call out Agent 1.4/Puppet 4.4 in the 5.x branch.